packages/dx-grid-core/src/plugins/sorting-state/helpers.js

```diff
@@ -1,4 +1,25 @@
+const unique = arr => [...new Set(arr)];
```

Comment:
Why we use `new Set()`?

Reply:
`Set` returns an array that contains only unique elements.